### ISSUE TITLE
Black validation images. 

### ISSUE THREAD

**Issue by Tauseefahmed1451** [huggingface/diffusers#6666]
### Describe the bug

Hello,  @yiyixuxu , @sayakpaul 
I am trying to train controlnet for sdxl. I have followed every step as instructed in Readme_sdxl in examples/controlnet. The pipeline runs fine, the data and model is loaded normally and the training runs smooth, but the validation images I get over on wandb are black
![image](https://github.com/huggingface/diffusers/assets/145332470/22abb5ba-f9ec-416f-9243-d36acbd77558)
just to verify I ran the sd1.5 pipeline on same dataset and all was ok. 
I changed LR, validation steps and changed all possible variables. I changed the images, checked for safety checkers, changed my dataset but still no improvement.
to note: I let one of my runs complete and the model is generating results (very very poor) that are not black. 

### Reproduction

export MODEL_DIR="stabilityai/stable-diffusion-xl-base-1.0"
export OUTPUT_DIR="models/diffusers/examples/controlnet/results"

accelerate launch train_controlnet_sdxl.py \
 --pretrained_model_name_or_path=$MODEL_DIR \
 --output_dir=$OUTPUT_DIR \
 --train_data_dir="models/diffusers/examples/controlnet/frontback"\
 --conditioning_image_column=conditioning_image \
 --image_column=image \
 --caption_column=text \
 --mixed_precision="fp16" \
 --resolution=1024 \
 --learning_rate=1e-5 \
 --max_train_steps=12000 \
--checkpointing_steps=1000 \
 --validation_image "./10.png" "./12.png" \
 --validation_prompt "gas_back" \
 --validation_steps=100 \
 --train_batch_size=4 \
 --gradient_accumulation_steps=4 \
 --report_to="wandb" \
 --seed=42

### System Info

TrainML
A6000 48 GB
100GB storage
Python3

### Who can help?

 @yiyixuxu @sayakpaul

**Comment by Tauseefahmed1451** [comment]
Checking, thanks.
